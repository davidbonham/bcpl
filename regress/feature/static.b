GET "libhdr"
 
STATIC $( count; one = 1  $)
 
LET start() BE $(
   count +:= 1
   writen(count)
   newline()
   writen(1)
$)

