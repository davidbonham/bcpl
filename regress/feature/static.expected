-- stdout -------------------------------------------
1
1

-- stderr -------------------------------------------

